TestAbortException/TestAbortException/UnitTest1.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TestAbortException
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+            TestMethod1();
```

Comment:
This will cause stackoverflow. How is this test case required for accessibility testing?